app/src/components/template/Dashboard.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import clsx from 'clsx';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { Drawer, AppBar, Toolbar, List, Typography, Divider, IconButton, Container, Link } from '@material-ui/core';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import MenuIcon from '@material-ui/icons/Menu';
+import { mainListItems } from './listItems';
+import { Styles as OBStyles } from '../../styles/OrderBookSnapshot';
+import OrderBookSnapshot from '../OrderBookSnapshot';
+import { Styles } from '../../styles/Dashboard';
+
+/**
+ * Returns copyright component for footer
+ */
+const Copyright = () => {
+    const classes = Styles();
+    return (
+        <Typography variant="body2" color="textSecondary" align="center" className={classes.paddingCopyright}>

```

Comment:
stuff like double quotes should be resolved by using the linting with our `eslintrc.json`. Your formatter is clearly not correct. I also encourage prop values to be set in curly braces, it's cleaner:

`variant={'body2'}`

Reply:
material UI demos uses like this. I didnt know that it wasnt correct. I have eslint extension installed, still isnt showing as a problem.